src/lib/containers/widgets/facet-nav/FacetNavPanel.tsx

```diff
@@ -424,100 +406,164 @@ const FacetNavPanel: React.FunctionComponent<FacetNavPanelProps> = ({
     )
   } else {
     return (
-      <div className={`FacetNavPanel${isExpanded ? '--expanded' : ''}`}>
-        <div className="FacetNavPanel__title">
-          <span className="FacetNavPanel__title__name">
-            {unCamelCase(facetToPlot.columnName, facetAliases)}
-          </span>
-          {isLoading && (
-            <span style={{ marginLeft: '2px' }} className={'spinner'} />
+      <>
+        <Modal
+          animation={false}
+          show={showModal}
+          onHide={() => setShowModal(false)}
+        >
+          <Modal.Header closeButton={true}>
+            <Modal.Title>{plotTitle ?? ''}</Modal.Title>
+          </Modal.Header>
+          <Modal.Body>
+            <FacetNavPanel {...props} isModalView={true} />
```

Comment:
Kinda strange to call itself, but we can easily recreate the same panel here (with `isModalView` set to true)